src/app.module.ts

```diff
@@ -57,6 +58,8 @@ export class AppModule implements NestModule {
       module: AppModule,
       imports: [
         // features
+        // TODO: Rely on configFactory to enable/disable features
+        AlertsModule,
```

Comment:
I will make this conditional in a follow-up PR as the `register` function will need to be made asyncronous - `configFactory` is a promise. It will otherwise affect a lot more than just the alerts domain.

Reply:
It's better to leave out than to have a follow up PR on this. Leaving the `AlertsModule` included in the `AppModule` would make it available to production if released.

Reply:
Good point. I will address this in a secondary PR. I've removed `AlertsModule` in 4c6e1c4f168e8008473165a0c10430e399bd30a5.